packages/native-stack/src/views/NativeStackView.tsx

```diff
@@ -151,6 +153,9 @@ const styles = StyleSheet.create({
   container: {
     flex: 1,
   },
+  contentContainer: {
```

Comment:
i created a separate style for this to express the intent more clearly but i don't see an issue with just reusing container, alternatively